### ISSUE TITLE
Allow coolcom to combine global and user modes

### ISSUE THREAD

**Issue by gmt2001** [gmt2001/PhantomBot-1#8]
Allow coolcom to have dual global/user cooldowns

For example:
```
CoolCom !hello global=10 user=20
User A uses !hello
User B can use !hello in 10 minutes (global)
User A must wait 20 minutes to use !hello again (user)
```

**Comment by Sartharon** [comment]
I'm currently working on this and caught this thought:
Currently, a command is either globally on a cooldown or on a cooldown per user.

The actual logic in the commandCoolDown.js I have finished, but further changes to **command event** in init.js need to be done to support the two different cooldown methods _at the same time_ (maybe even to the database). Plus, the Panel needs to be updated as well to reflect the two types of cooldown. Currently, there is only a checkbox to select **globally** or not and a text box for the **duration** of the cooldown.

But before I proceed, I'd just wanted to check if that was really your intention.
